feature-libs/cart/saved-cart/occ/adapters/occ-saved-cart.adapter.ts

```diff
@@ -40,7 +40,7 @@ export class OccSavedCartAdapter implements SavedCartAdapter {
   saveCart(
     userId: string,
     cartId: string,
-    saveCartName?: string,
+    saveCartName: string,
```

Comment:
let's keep this optional as the api does not require to have the save cart name. Even if it was for the wires, we can make the form required, but make our services have the optional param